premis3.owl

```diff
@@ -0,0 +1,672 @@
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/Statute">
+    <rdfs:label>Statute</rdfs:label>
+    <rdfs:comment>A law that grants or revokes rights, such as laws governing privacy or orphan
+      works.</rdfs:comment>
+    <rdfs:subClassOf rdf:resource="http://www.loc.gov/premis/rdf/v3/RightsBasis"/>
+    <owl:disjointWith rdf:resource="http://www.loc.gov/premis/rdf/v3/Copyright"/>
+    <owl:disjointWith rdf:resource="http://www.loc.gov/premis/rdf/v3/License"/>
+    <owl:disjointWith rdf:resource="http://www.loc.gov/premis/rdf/v3/InstitutionalPolicy"/>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/StorageLocation">
+    <rdfs:label>Storage location</rdfs:label>
+    <rdfs:comment>Information needed to retrieve a physical item from its physical storage location or a file
+      from the storage system, or to access a bitstream within a file.</rdfs:comment>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+    <rdfs:seeAlso rdf:resource="http://id.loc.gov/vocabulary/preservation/contentLocationType"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="http://www.loc.gov/premis/rdf/v3/StorageMedium">
+    <rdfs:label>Storage medium</rdfs:label>
+    <rdfs:comment>The physical medium on which the object  is stored (e.g., magnetic tape, hard disk, CD-ROM,
+      DVD).</rdfs:comment>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+    <rdfs:seeAlso rdf:resource="http://id.loc.gov/vocabulary/preservation/storageMedium"/>
+  </owl:Class>
+
+
+  <!-- Object Properties -->
+
+  <owl:ObjectProperty rdf:about="http://www.loc.gov/premis/rdf/v3/act">
+    <rdfs:label>act</rdfs:label>
+    <rdfs:domain rdf:resource="http://www.loc.gov/premis/rdf/v3/Rule"/>
+    <rdfs:range rdf:resource="http://www.loc.gov/premis/rdf/v3/Action"/>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+  </owl:ObjectProperty>
+
+  <owl:ObjectProperty rdf:about="http://www.loc.gov/premis/rdf/v3/allows">
+    <rdfs:label>allows</rdfs:label>
+    <rdfs:domain rdf:resource="http://www.loc.gov/premis/rdf/v3/RightsBasis"/> 
+    <rdfs:range rdf:resource="http://www.loc.gov/premis/rdf/v3/Rule"/>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+  </owl:ObjectProperty>
+
+  <owl:ObjectProperty rdf:about="http://www.loc.gov/premis/rdf/v3/governs">
+    <rdfs:label>governs</rdfs:label>
+    <owl:domain rdf:resource="http://www.loc.gov/premis/rdf/v3/RightsStatus"/>
+    <owl:range rdf:resource="http://www.loc.gov/premis/rdf/v3/Object"/>
+    <rdfs:isDefinedBy rdf:resource="http://www.loc.gov/premis/rdf/v3/"/>
+  </owl:ObjectProperty> 
+
+  <owl:ObjectProperty rdf:about="http://www.loc.gov/premis/rdf/v3/hasBasis">
+    <rdfs:label>has basis</rdfs:label>
```

Comment:
In fact, the property was meant to point from an instance of RightsStatus to an instance of RightsBasis. It was intended for cases where an Object is governed by two different rights basis, with different applicable dates attached to the Rights Status, which had to be associated with the right Rights basis.